### PR TITLE
Prepare XCP-D versions of atlases

### PR DESCRIPTION
**Pull request by tsalo**
Changes proposed:

- Rename atlases to use correct entity order (see #2).

### CODE REVIEW

File: schaefer_cifti/tpl-MNI152NLin6Asym_atlas-Schaefer2018v0143Merged_res-01_desc-1000Parcels_dseg.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        "LongName": "Node label from Schaefer 17-network atlas"
+    },
+    "label_7network": {
+        "Description": "The label of the node as it appears in the canonical Schaefer 7-network atlas.",
+        "LongName": "Node label from Schaefer 7-network atlas"
+    },
+    "network_label_17network": {
+        "Description": "The name of the network this node is assigned to in the Schaefer 17-network atlas.",
+        "LongName": "Network label from Schaefer 17-network atlas"
+    }
```
Comment (tsalo) [inline schaefer_cifti/tpl-MNI152NLin6Asym_atlas-Schaefer2018v0143Merged_res-01_desc-1000Parcels_dseg.json]: Descriptions of custom columns.

File: schaefer_cifti/tpl-fsLR_hemi-L_den-32k_desc-nomedialwall_dparc.label.gii [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mattcieslak) [inline schaefer_cifti/tpl-fsLR_hemi-L_den-32k_desc-nomedialwall_dparc.label.gii]: Here's one of the giftis that ended up being tracked in git

File: tpl-fsLR_atlas-4S1052Parcels_den-91k_dseg.dlabel.nii [hunk 3]
```diff
@@ -1 +1 @@
-.git/annex/objects/52/VG/MD5E-s1183480--4cc39dd835a341c2111de7028e649067.nii/MD5E-s1183480--4cc39dd835a341c2111de7028e649067.nii
\ No newline at end of file
+.git/annex/objects/3q/MX/MD5E-s1133528--408c04f17819ef7c8da933a61a2dc442.nii/MD5E-s1133528--408c04f17819ef7c8da933a61a2dc442.nii
```
Comment (mattcieslak) [inline tpl-fsLR_atlas-4S1052Parcels_den-91k_dseg.dlabel.nii]: this one is appropriately in git annex

File: tpl-fsLR_atlas-4S252Parcels_dseg.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
     ],
     "Density": {
         "32k": "32492 vertices per hemisphere. No subcortical data.",
-        "91k": "96854 vertices. 29696 surface vertices in left hemisphere. 29716 surface vertices in right hemisphere. 31870 subcortical voxels, at 2 mm isotropic resolution."
+        "91k": "91282 vertices. 29696 surface vertices in left hemisphere. 29716 surface vertices in right hemisphere. 31870 subcortical voxels, at 2 mm isotropic resolution."
```
Comment (mattcieslak) [inline tpl-fsLR_atlas-4S252Parcels_dseg.json:17]: so useful!

**Review by mattcieslak — COMMENTED** [review]
Are you a fan of the Schaefer2018v01431000Parcels desc? I think it's pretty tough to read. What about Schaefer2018v0143x1000Parcels?

Also, I think the gifti files got sent to git instead of git annex. This probably isn't a big deal, but if you think there will be lots of changes to these files we should switch them over to git annex

**Comment by tsalo** [comment]
Okay, I _think_ I've made the requested changes.

File: .gitattributes [hunk 5]
```diff
@@ -1,3 +1,5 @@
 * annex.backend=MD5E
 **/.git* annex.largefiles=nothing
 * annex.largefiles=((mimeencoding=binary)and(largerthan=0))
+**/*.label.gii annex.largefiles=(anything)
```
Comment (mattcieslak) [inline .gitattributes:4]: looks great

File: schaefer_cifti/run_docker.sh [hunk 6]
```diff
@@ -0,0 +1,6 @@
+docker run -it \
```
Comment (mattcieslak) [inline schaefer_cifti/run_docker.sh:1]: What does this do?

File: schaefer_cifti/tpl-MNI152NLin6Asym_atlas-SubcorticalMerged_res-02_desc-Schaefer2018v0143x300Parcels_labellist.txt [hunk 7]
```diff
@@ -0,0 +1,104 @@
+LH-Pu
+301 11 193 193 255
```
Comment (mattcieslak) [inline schaefer_cifti/tpl-MNI152NLin6Asym_atlas-SubcorticalMerged_res-02_desc-Schaefer2018v0143x300Parcels_labellist.txt:2]: are these the colors?

**Review by mattcieslak — APPROVED** [review]

File: schaefer_cifti/run_docker.sh (hunk 6 above)
Comment (tsalo) [reply schaefer_cifti/run_docker.sh:1]: I had to run a Docker image with ANTs interactively to download the brain mask image with templateflow and run `01_downsample_subcortical.sh`

File: schaefer_cifti/tpl-MNI152NLin6Asym_atlas-SubcorticalMerged_res-02_desc-Schaefer2018v0143x300Parcels_labellist.txt (hunk 7 above)
Comment (tsalo) [reply schaefer_cifti/tpl-MNI152NLin6Asym_atlas-SubcorticalMerged_res-02_desc-Schaefer2018v0143x300Parcels_labellist.txt:2]: Yeah, the second through fourth numbers are the colors, with the fifth being the alpha value.

**Pull request merged**